test/v3/removeLiquidity.integration.test.ts

```diff
@@ -471,40 +476,73 @@ async function putPoolIntoRecoveryMode(
     poolState: PoolState,
     authorizedAddress: Address,
 ) {
+    // get the actionId for the enableRecoveryMode function
+    const actionId = await client.readContract({
+        address: VAULT_ADMIN[chainId],
+        abi: vaultAdminAbi,
+        functionName: 'getActionId',
+        args: [toFunctionSelector('function enableRecoveryMode(address)')],
+    });
+
```

Comment:
Think it is better to programmatically compute the actionId instead of storing it as a constant. Was a bit of a trip figuring out the actionId needed to be updated :sweat_smile: